src/extensionsIntegrated/CSSColorPreview/main.js

```diff
@@ -33,6 +33,8 @@ define(function (require, exports, module) {
         Editor              = require("editor/Editor").Editor,
```

Comment:
I have rebased this pr as there were some other code changes, just a heads up if u face any issues while pushing